### ISSUE TITLE
Cart should not store the variable 'tax_display_cart' in it's constructor

### ISSUE THREAD

**Issue by ewijdemans** [woocommerce/woocommerce#26242]
**Describe the bug**
The cart (WC_Cart) constructor stores a local variable named 'tax_display_cart'. This variable is used by the method 'display_prices_including_tax'. See below:

```
	public function __construct() {
		$this->session          = new WC_Cart_Session( $this );
		$this->fees_api         = new WC_Cart_Fees( $this );
		$this->tax_display_cart = $this->is_tax_displayed();

		// Register hooks for the objects.
		$this->session->init();

		add_action( 'woocommerce_add_to_cart', array( $this, 'calculate_totals' ), 20, 0 );
		add_action( 'woocommerce_applied_coupon', array( $this, 'calculate_totals' ), 20, 0 );
		add_action( 'woocommerce_cart_item_removed', array( $this, 'calculate_totals' ), 20, 0 );
		add_action( 'woocommerce_cart_item_restored', array( $this, 'calculate_totals' ), 20, 0 );
		add_action( 'woocommerce_check_cart_items', array( $this, 'check_cart_items' ), 1 );
		add_action( 'woocommerce_check_cart_items', array( $this, 'check_cart_coupons' ), 1 );
		add_action( 'woocommerce_after_checkout_validation', array( $this, 'check_customer_coupons' ), 1, 2 );
	}
```

If any plugin manipulates the users' VAT exempt via the 'set_is_vat_exempt' method of WC_Customer (in my case during the 'woocommerce_checkout_update_order_review'-hook), the 'display_prices_including_tax' method of WC_Cart will now ignore it's value as it's stored locally in the construct of WC_Cart.

Basically what the 'display_prices_including_tax' method does is executing this below piece of code twice, once in the construct of WC_Cart, and once on the spot.

`
$this->get_customer() && $this->get_customer()->get_is_vat_exempt()
`

This results in the customers' VAT exempt value to be ignored if manipulated between the execution construct and the execution of the 'display_prices_including_tax' method of WC_Cart.

My suggestion is to alter the 'display_prices_including_tax' function to use the actual value, like this:

```
public function display_prices_including_tax() {
		$customer_exempt = $this->get_customer()
            ? $this->get_customer()->get_is_vat_exempt()
            : 'incl' === $this->tax_display_cart;

		return apply_filters( 'woocommerce_cart_' . __FUNCTION__, !$customer_exempt );
	}
```

I can reproduce this bug WITH a plugin enabled that manipulates the VAT exempt of the customer so therefor I cannot report this issue without it.

**WordPress Environment**
<details>
```
`
### WordPress Environment ###

WordPress address (URL): https://www.infentorides.test
Site address (URL): https://www.infentorides.test
WC Version: 4.0.1
REST API Version: ✔ 1.0.7
WC Blocks Version: ✔ 2.5.14
Action Scheduler Version: ✔ 3.1.4
WC Admin Version: ✔ 1.0.3
Log Directory Writable: ✔
WP Version: ❌ 5.3.2 - There is a newer version of WordPress available (5.4)
WP Multisite: –
WP Memory Limit: 256 MB
WP Debug Mode: ✔
WP Cron: ✔
Language: en_US
External object cache: –

### Server Environment ###

Server Info: nginx/1.17.2
PHP Version: 7.3.7
PHP Post Max Size: 100 MB
PHP Time Limit: 30
PHP Max Input Vars: 1000
cURL Version: 7.65.3
OpenSSL/1.0.2s

SUHOSIN Installed: –
MySQL Version: 5.7.29
Max Upload Size: 100 MB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ❌ wp_remote_post() failed. Contact your hosting provider.
Remote Get: ✔

### Database ###

WC Database Version: 4.0.1
WC Database Prefix: wp_
Total Database Size: 451.95MB
Database Data Size: 305.33MB
Database Index Size: 146.62MB
wp_woocommerce_sessions: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_api_keys: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_attribute_taxonomies: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_downloadable_product_permissions: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_woocommerce_order_items: Data: 2.52MB + Index: 1.52MB + Engine InnoDB
wp_woocommerce_order_itemmeta: Data: 17.55MB + Index: 13.03MB + Engine InnoDB
wp_woocommerce_tax_rates: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_woocommerce_tax_rate_locations: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_shipping_zones: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_woocommerce_shipping_zone_locations: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_shipping_zone_methods: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_woocommerce_payment_tokens: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_payment_tokenmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_log: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
monolog: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wp_abandoned_cart_aelia_currency: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_actionscheduler_actions: Data: 0.33MB + Index: 0.41MB + Engine InnoDB
wp_actionscheduler_claims: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_actionscheduler_groups: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_actionscheduler_logs: Data: 0.25MB + Index: 0.22MB + Engine InnoDB
wp_ac_abandoned_cart_history: Data: 19.55MB + Index: 0.36MB + Engine InnoDB
wp_ac_email_templates: Data: 0.11MB + Index: 0.00MB + Engine InnoDB
wp_ac_guest_abandoned_cart_history: Data: 1.52MB + Index: 0.14MB + Engine InnoDB
wp_ac_link_clicked_email: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_ac_notifications: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_ac_notifications_meta: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_ac_opened_emails: Data: 0.14MB + Index: 0.00MB + Engine InnoDB
wp_ac_sent_history: Data: 0.20MB + Index: 0.09MB + Engine InnoDB
wp_ac_tiny_urls: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_aelia_dismissed_messages: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_aelia_exchange_rates_history: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_aepc_custom_audiences: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_aepc_logs: Data: 0.23MB + Index: 0.00MB + Engine InnoDB
wp_commentmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_comments: Data: 11.52MB + Index: 11.09MB + Engine InnoDB
wp_elfsight_instagram_feed_cache: Data: 1.52MB + Index: 0.00MB + Engine InnoDB
wp_elfsight_instagram_feed_throttle: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_elfsight_instagram_feed_widgets: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_failed_jobs: Data: 1.52MB + Index: 0.00MB + Engine InnoDB
wp_gf_addon_feed: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_gf_draft_submissions: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_gf_entry: Data: 7.52MB + Index: 1.88MB + Engine InnoDB
wp_gf_entry_meta: Data: 5.52MB + Index: 9.55MB + Engine InnoDB
wp_gf_entry_notes: Data: 0.05MB + Index: 0.03MB + Engine InnoDB
wp_gf_form: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_gf_form_meta: Data: 0.09MB + Index: 0.00MB + Engine InnoDB
wp_gf_form_revisions: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_gf_form_view: Data: 0.20MB + Index: 0.17MB + Engine InnoDB
wp_instapage_debug: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_instapage_options: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_instapage_pages: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_links: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mailchimp_carts: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_mailchimp_jobs: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_options: Data: 2.20MB + Index: 0.20MB + Engine InnoDB
wp_pmxi_files: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_pmxi_history: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_pmxi_images: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_pmxi_imports: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_pmxi_posts: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_pmxi_templates: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_postmeta: Data: 155.61MB + Index: 88.78MB + Engine InnoDB
wp_posts: Data: 49.52MB + Index: 3.92MB + Engine InnoDB
wp_pum_subscribers: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_queue: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_redirection_404: Data: 0.17MB + Index: 0.14MB + Engine InnoDB
wp_redirection_groups: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_redirection_items: Data: 0.02MB + Index: 0.09MB + Engine InnoDB
wp_redirection_logs: Data: 0.02MB + Index: 0.08MB + Engine InnoDB
wp_rg_form: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_rg_form_meta: Data: 0.08MB + Index: 0.00MB + Engine InnoDB
wp_rg_form_view: Data: 0.08MB + Index: 0.06MB + Engine InnoDB
wp_rg_incomplete_submissions: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_rg_lead: Data: 3.52MB + Index: 0.36MB + Engine InnoDB
wp_rg_lead_detail: Data: 1.52MB + Index: 2.28MB + Engine InnoDB
wp_rg_lead_detail_long: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_rg_lead_meta: Data: 2.52MB + Index: 3.44MB + Engine InnoDB
wp_rg_lead_notes: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_termmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_terms: Data: 0.20MB + Index: 0.20MB + Engine InnoDB
wp_term_relationships: Data: 1.52MB + Index: 1.52MB + Engine InnoDB
wp_term_taxonomy: Data: 0.27MB + Index: 0.23MB + Engine InnoDB
wp_usermeta: Data: 8.52MB + Index: 2.00MB + Engine InnoDB
wp_users: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_wc_admin_notes: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wc_admin_note_actions: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_category_lookup: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wc_customer_lookup: Data: 1.52MB + Index: 0.67MB + Engine InnoDB
wp_wc_download_log: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_order_coupon_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_order_product_lookup: Data: 2.52MB + Index: 1.53MB + Engine InnoDB
wp_wc_order_stats: Data: 1.52MB + Index: 0.92MB + Engine InnoDB
wp_wc_order_tax_lookup: Data: 1.52MB + Index: 0.64MB + Engine InnoDB
wp_wc_product_meta_lookup: Data: 0.06MB + Index: 0.09MB + Engine InnoDB
wp_wc_tax_rate_classes: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_webhooks: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woochimp_scheduled_events: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_woocommerce_icepay_pminfo: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_woocommerce_icepay_pmrawdata: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_woocommerce_icepay_transactions: Data: 0.39MB + Index: 0.00MB + Engine InnoDB
wp_yoast_seo_links: Data: 0.33MB + Index: 0.14MB + Engine InnoDB
wp_yoast_seo_meta: Data: 0.16MB + Index: 0.00MB + Engine InnoDB

### Post Type Counts ###

acf-field: 125
acf-field-group: 40
attachment: 2886
construction: 359
creation: 118
customize_changeset: 1
faq: 139
fl-builder-template: 62
in_the_news: 148
instagram: 26
instruction: 17
nav_menu_item: 140
oembed_cache: 1
page: 207
part: 407
polylang_mo: 5
popup: 2
popup_theme: 7
post: 243
product: 334
product-feature: 41
revision: 2124
shop_coupon: 37
shop_order: 10395
shop_order_refund: 166
testimonial: 124

### Security ###

Secure connection (HTTPS): ✔
Hide errors from visitors: ✔

### Active Plugins (41) ###

Polylang Pro: by WP SYNTEX – 2.7.1
Gravity Forms: by rocketgenius – 2.4.17
Redux Framework: by Team Redux – 3.6.18
Advanced Custom Fields: Gravity Forms Field: by @adam_pope of @stormuk – 1.1.0
Advanced Custom Fields PRO: by Elliot Condon – 5.8.6
Beaver Builder Plugin (Pro Version): by The Beaver Builder Team – 2.3.2.5
Booster Plus for WooCommerce: by Algoritmika Ltd – 3.4.4 – Not tested with the active version of WooCommerce
DsgnWrks Instagram Importer: by DsgnWrks – 2.1.1
Flexible Checkout Fields: by WP Desk – 2.4.7 – Not tested with the active version of WooCommerce
Gravity Forms: Add Class To Submit Button: by poweredbycoffee – 1.0
Integrate Gravity Forms + Polylang: by Philip Ebels – 0.3
Gravity Forms Mailchimp Add-On: by Gravity Forms – 4.7
ICEPAY PaymentResponse Module: by ICEPAY – 2.3.6 – Not tested with the active version of WooCommerce
Kraken Image Optimizer: by Karim Salman – 2.6.4
LiveChat WooCommerce: by LiveChat – 1.5.1 – Not tested with the active version of WooCommerce
Loco Translate: by Tim Whitlock – 2.3.3
Mailchimp for WooCommerce: by Mailchimp – 2.3.6 – Not tested with the active version of WooCommerce
Polylang for WooCommerce: by WP SYNTEX – 1.4 – Not tested with the active version of WooCommerce
RAoW: by RAoW – 0.1.2
Redirection: by John Godley – 4.7.1
Regenerate Thumbnails: by Alex Mills (Viper007Bond) – 3.1.3
Rewrite Rules Inspector: by Daniel Bachhuber
Automattic – 1.2.1

Simple Custom Post Order: by Colorlib – 2.4.9
SVG Support: by Benbodhi – 2.3.18
WooCommerce - Order Export: by Apin Chaudhary – 3.1.1 – Not tested with the active version of WooCommerce
Abandoned Cart Pro for WooCommerce: by Tyche Softwares – 8.3.0 – Not tested with the active version of WooCommerce
WooCommerce Extended Coupon Features PRO: by Soft79 – 3.1.4.1 – Not tested with the active version of WooCommerce
WooCommerce Email Customizer: by cxThemes – 3.32 – Not tested with the active version of WooCommerce
WooCommerce EU VAT Assistant: by Aelia – 1.13.6.200415 – Not tested with the active version of WooCommerce
Booster for WooCommerce: by Algoritmika Ltd – 4.8.0 – Not tested with the active version of WooCommerce
WooCommerce Shipment Tracking: by WooCommerce – 1.6.4 – Not tested with the active version of WooCommerce
WooCommerce Shipping Estimates: by SkyVerge – 2.1.0 – Not tested with the active version of WooCommerce
WooCommerce: by Automattic – 4.0.1
Yoast SEO: by Team Yoast – 13.5
WP All Import Pro: by Soflyy – 4.5.9
WP Mail Options: by Soli – 0.2.3
WP Mail SMTP: by WPForms – 1.9.0
WP-PageNavi: by Lester 'GaMerZ' Chan – 2.93.3
WP All Import - ACF Add-On: by Soflyy – 3.2.5
WP All Import - WooCommerce Add-On Pro: by Soflyy – 3.1.1 – Not tested with the active version of WooCommerce
Yoast SEO: WooCommerce: by Team Yoast – 6.3 – Not tested with the active version of WooCommerce

### Inactive Plugins (17) ###

Advanced Custom Fields: Read only: by Quantity Digital – 0.5.0
Easy WP SMTP: by wpecommerce
alexanderfoxc – 1.3.9.2

Facebook for WooCommerce: by Facebook – 1.10.2 – Not tested with the active version of WooCommerce
Monkeyman Rewrite Analyzer: by Jan Fabry – 1.0
Permalink Manager Pro: by Maciej Bis – 2.2.5 – Not tested with the active version of WooCommerce
Pixel Caffeine: by AdEspresso – 2.1.1
PixelYourSite: by PixelYourSite – 7.1.7 – Not tested with the active version of WooCommerce
Polylang Sync: by Jörn Lund – 0.0.14
Premmerce Permalink Manager for WooCommerce (Premium): by premmerce – 2.1.12 – Not tested with the active version of WooCommerce
Pronamic Pay: by Pronamic – 6.1.1
Pronamic Pay Adyen Add-On: by Pronamic – 1.0.0
UpdraftPlus - Backup/Restore: by UpdraftPlus.Com
DavidAnderson – 1.16.23

WooChimp: by RightPress – 2.2.4 – Not tested with the active version of WooCommerce
WooCommerce Admin: by WooCommerce – 1.0.3 – Not tested with the active version of WooCommerce
WooCommerce Google Analytics Pro: by SkyVerge – 1.3.3 – Not tested with the active version of WooCommerce
WooCommerce PayPal Checkout Gateway: by WooCommerce – 1.6.21 – Not tested with the active version of WooCommerce
WP Rocket: by WP Media – 2.10.7

### Dropin Plugins (1) ###

advanced-cache.php: advanced-cache.php

### Settings ###

API Enabled: ✔
Force SSL: –
Currency: USD ($)
Currency Position: left
Thousand Separator: ,
Decimal Separator: .
Number of Decimals: 2
Taxonomies: Product Types: external (external)
grouped (grouped)
simple (simple)
variable (variable)

Taxonomies: Product Visibility: exclude-from-catalog (exclude-from-catalog)
exclude-from-search (exclude-from-search)
featured (featured)
outofstock (outofstock)
rated-1 (rated-1)
rated-2 (rated-2)
rated-3 (rated-3)
rated-4 (rated-4)
rated-5 (rated-5)

Connected to WooCommerce.com: –

### WC Pages ###

Shop base: #3552 - /shop/
Cart: #3553 - /cart/
Checkout: #3554 - /checkout/
My account: #3555 - /account/
Terms and conditions: #11409 - /general-terms/

### Theme ###

Name: Infento
Version: 1.0
Author URL: http://www.raow.nl
Child Theme: ❌ – If you are modifying WooCommerce on a parent theme that you did not build personally we recommend using a child theme. See: How to create a child theme
WooCommerce Support: ✔

### Templates ###

Archive Template: Your theme has a woocommerce.php file
you will not be able to override the woocommerce/archive-product.php custom template since woocommerce.php has priority over archive-product.php. This is intended to prevent display issues.

Overrides: infento/woocommerce/cart/cart-shipping.php version 2.5.0 is out of date. The core version is 3.6.0
infento/woocommerce/cart/cart.php version 3.3.0 is out of date. The core version is 3.8.0
infento/woocommerce/cart/mini-cart.php version - is out of date. The core version is 3.7.0
infento/woocommerce/cart/proceed-to-checkout-button.php
infento/woocommerce/checkout/form-checkout.php version 2.3.0 is out of date. The core version is 3.5.0
infento/woocommerce/checkout/form-coupon.php version 3.3.0 is out of date. The core version is 3.4.4
infento/woocommerce/checkout/form-pay.php
infento/woocommerce/checkout/form-shipping.php version 3.0.9 is out of date. The core version is 3.6.0
infento/woocommerce/checkout/payment-method.php version 2.3.0 is out of date. The core version is 3.5.0
infento/woocommerce/checkout/payment.php
infento/woocommerce/checkout/review-order.php version 2.3.0 is out of date. The core version is 3.8.0
infento/woocommerce/checkout/terms.php version 3.1.1 is out of date. The core version is 3.4.0
infento/woocommerce/checkout/thankyou.php version 3.0.0 is out of date. The core version is 3.7.0
infento/woocommerce/content-product.php version 3.0.0 is out of date. The core version is 3.6.0
infento/woocommerce/content-single-product.php version - is out of date. The core version is 3.6.0
/Users/erikwijdemans/Documents/Development/sites/infentorides/www/wp-content/plugins/woocommerce-email-control/templates/emails/vanilla/admin-cancelled-order.php
/Users/erikwijdemans/Documents/Development/sites/infentorides/www/wp-content/plugins/woocommerce-email-control/templates/emails/vanilla/admin-failed-order.php
/Users/erikwijdemans/Documents/Development/sites/infentorides/www/wp-content/plugins/woocommerce-email-control/templates/emails/vanilla/admin-new-order.php
/Users/erikwijdemans/Documents/Development/sites/infentorides/www/wp-content/plugins/woocommerce-email-control/templates/emails/vanilla/customer-completed-order.php
/Users/erikwijdemans/Documents/Development/sites/infentorides/www/wp-content/plugins/woocommerce-email-control/templates/emails/vanilla/customer-invoice.php
/Users/erikwijdemans/Documents/Development/sites/infentorides/www/wp-content/plugins/woocommerce-email-control/templates/emails/vanilla/customer-new-account.php
/Users/erikwijdemans/Documents/Development/sites/infentorides/www/wp-content/plugins/woocommerce-email-control/templates/emails/vanilla/customer-note.php
/Users/erikwijdemans/Documents/Development/sites/infentorides/www/wp-content/plugins/woocommerce-email-control/templates/emails/vanilla/customer-on-hold-order.php
/Users/erikwijdemans/Documents/Development/sites/infentorides/www/wp-content/plugins/woocommerce-email-control/templates/emails/vanilla/customer-processing-order.php
/Users/erikwijdemans/Documents/Development/sites/infentorides/www/wp-content/plugins/woocommerce-email-control/templates/emails/vanilla/customer-refunded-order.php
/Users/erikwijdemans/Documents/Development/sites/infentorides/www/wp-content/plugins/woocommerce-email-control/templates/emails/vanilla/customer-reset-password.php version 3.7.0 is out of date. The core version is 4.0.0
/Users/erikwijdemans/Documents/Development/sites/infentorides/www/wp-content/plugins/woocommerce-email-control/templates/emails/vanilla/email-addresses.php version 3.5.4 is out of date. The core version is 3.9.0
/Users/erikwijdemans/Documents/Development/sites/infentorides/www/wp-content/plugins/woocommerce-email-control/templates/emails/vanilla/email-customer-details.php
/Users/erikwijdemans/Documents/Development/sites/infentorides/www/wp-content/plugins/woocommerce-email-control/templates/emails/vanilla/email-downloads.php
/Users/erikwijdemans/Documents/Development/sites/infentorides/www/wp-content/plugins/woocommerce-email-control/templates/emails/vanilla/email-footer.php
/Users/erikwijdemans/Documents/Development/sites/infentorides/www/wp-content/plugins/woocommerce-email-control/templates/emails/vanilla/email-header.php version 2.4.0 is out of date. The core version is 4.0.0
/Users/erikwijdemans/Documents/Development/sites/infentorides/www/wp-content/plugins/woocommerce-email-control/templates/emails/vanilla/email-order-details.php
/Users/erikwijdemans/Documents/Development/sites/infentorides/www/wp-content/plugins/woocommerce-email-control/templates/emails/vanilla/email-order-items.php
/Users/erikwijdemans/Documents/Development/sites/infentorides/www/wp-content/plugins/woocommerce-email-control/templates/emails/vanilla/email-styles.php version 3.3.0 is out of date. The core version is 4.0.0
infento/woocommerce/global/quantity-input.php version 3.3.0 is out of date. The core version is 4.0.0
infento/woocommerce/order/order-details-customer.php version 3.0.0 is out of date. The core version is 3.4.4
infento/woocommerce/order/order-details.php version 3.0.0 is out of date. The core version is 3.7.0

Outdated Templates: ❌
					
					
						Learn how to update

### WC Pages Translations ###

Shop base: ✔
Cart: ✔
Checkout: ✔
My account: ✔

### Action Scheduler ###

Canceled: 4
Oldest: 2020-03-23 13:39:02 +0100
Newest: 2020-04-07 11:58:49 +0200

Complete: 1,045
Oldest: 2020-03-23 13:38:25 +0100
Newest: 2020-04-22 13:33:08 +0200

`
```
</details>

**Comment by claudiosanches** [comment]
Hi @ewijdemans,

Thank you for taking the time reporting this bug, we really appreciate your help. I can reproduce it on my end.

I took a different direction to fix it, since we need to keep backwards compatibility too, you can check the proposed fix in #26400.

**Comment by ewijdemans** [comment]
Hi Claudio,

Thanks for reviewing my bug report. I would say that this should indeed be
the approach.

Op do 7 mei 2020 om 05:13 schreef Claudio Sanches <notifications@github.com
